Runtime/Core/ProBuilderMeshFunction.cs

```diff
@@ -326,13 +350,14 @@ public void Refresh(RefreshMask mask = RefreshMask.All)
         void RefreshCollisions()
         {
             mesh.RecalculateBounds();
+            MeshCollider collider;
 
-            var meshCollider = GetComponent<MeshCollider>();
-
-            if (meshCollider != null)
+            if(gameObject.TryGetComponent<MeshCollider>(out collider))
             {
-                gameObject.GetComponent<MeshCollider>().sharedMesh = null;
-                gameObject.GetComponent<MeshCollider>().sharedMesh = mesh;
+                SerializationUtility.UnregisterDrivenProperty(this, collider, "m_Mesh");
+                SerializationUtility.RegisterDrivenProperty(this, collider, "m_Mesh");
+                collider.sharedMesh = null;
+                collider.sharedMesh = mesh;
```

Comment:
Is this double assignation required?

Reply:
It used to be a bug that this was required, it might no longer be necessary. I will check.